### PR TITLE
minor edits for emr on eks tutorial

### PR DESCRIPTION
**Pull request by dalbhanj**
Issue #, if available: https://github.com/aws-controllers-k8s/community/issues/1452
Original PR #: https://github.com/aws-controllers-k8s/community/pull/1453

Description of changes: minor edits to fix links and grammer

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/content/docs/tutorials/emr-on-eks-example.md [hunk 1]
```diff
@@ -352,17 +353,17 @@ kubectl logs ${CONTROLLER_POD} -n ${ACK_K8S_NAMESPACE}
...
         - $(AWS_ENDPOINT_URL)
-        **- --enable-development-logging
+        - --enable-development-logging
         - "true"
         - --log-level
-        - debug**
+        - debug
         - --resource-tags
         - $(ACK_RESOURCE_TAGS)
 ```
```
Comment (RedbackThomson) [inline docs/content/docs/tutorials/emr-on-eks-example.md:368]: Hmm I guess we didn't make the changes in the original PR, but really we shouldn't be touching deployments directly. Only by updating the Helm chart values and redeploying.

File: docs/content/docs/tutorials/emr-on-eks-example.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Run Spark jobs using EMR on EKS controller"
+title: "Run Spark jobs using ACK service controller for EMR on EKS"
```
[the remark this answers is not in the document]
Comment (dalbhanj) [reply docs/content/docs/tutorials/emr-on-eks-example.md:2]: fixed

File: docs/content/docs/tutorials/emr-on-eks-example.md (hunk 1 above)
Comment (dalbhanj) [reply docs/content/docs/tutorials/emr-on-eks-example.md:368]: This is mainly for troubleshooting purpose. I see end users using `kubectl` and not using `helm` for troubleshooting. Had a discussion with Amine on this [here](https://github.com/aws-controllers-k8s/community/pull/1453#pullrequestreview-1098158023)

**Review by A-Hilaly — APPROVED** [review]
/lgtm

**Pull request merged**